chall01/mdesta.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+# Map a char to Morse Code
+def encodeToMorse(x): 	
+	if x is 'a': 
+		return ".-"
+	elif x is 'b': 
+		return "-..."
+	elif x is 'c': 
+		return "-.-."
+	elif x is 'd': 
+		return "-.."
+	elif x is 'e': 
+		return "."
+	elif x is 'f': 
+		return "..-."
```

Comment:
If-forest, it's not really pythonic, and this should be avoided in all the language

You should have used Python Dict : https://docs.python-guide.org/writing/style/

```python
mydict = {'a': '.-', 'b': '-...', 'c': '-.-.', 'd': '-..', 'e': '.', 'f': '..-.', 'g': '--.', 'h': '....', 'i': '..', 'j': '.---', 'k': '-.-', 'l': '.-..', 'm': '--', 'n': '-.', 'o': '---', 'p': '.--.', 'q': '--.-', 'r': '.-.', 's': '...', 't': '-', 'u': '..-', 'v': '...-', 'w': '.--', 'x': '-..-', 'y': '-.--', 'z': '--..'}
```

Then `return mydict[x]`